### PR TITLE
Signed TLS certificates

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: pkg/install/2-removebootstrap.go [hunk 1]
```diff
@@ -135,6 +141,107 @@ func (i *Installer) removeBootstrap(ctx context.Context) error {
...
+			}
+
+			_, err = ocli.OperatorV1().IngressControllers("openshift-ingress-operator").Update(ic)
+			return err
+		})
+		if err != nil {
+			return err
+		}
+
+		// TODO: wait
```
Comment (mjudeikis) [inline pkg/install/2-removebootstrap.go]: I think we should aggregate all objects and have one wait loop below with `map[object]bool` so update would be sent to API and waits are parallelized.

File: deploy/rp-development.json [hunk 2]
```diff
@@ -35,6 +35,41 @@
...
+                    }
+                ]
+            },
+            "name": "[concat(parameters('keyvaultPrefix'), '-clusters')]",
+            "type": "Microsoft.KeyVault/vaults",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "vault": "clusters"
+            },
+            "apiVersion": "2016-10-01"
```
Comment (mjudeikis) [inline deploy/rp-development.json]: How so we use `v7.0` client SDK and this version?

File: deploy/rp-development.json [hunk 3]
```diff
@@ -35,6 +35,41 @@
...
+                        "tenantId": "[subscription().tenantId]",
+                        "objectId": "[parameters('adminObjectId')]"
+                    }
+                ]
+            },
+            "name": "[concat(parameters('keyvaultPrefix'), '-clusters')]",
+            "type": "Microsoft.KeyVault/vaults",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "vault": "clusters"
```
[the remark this answers is not in the document]
Comment (mjudeikis) [reply deploy/rp-development.json]: ok, after reading all PR not it is clear

[the remark this answers is not in the document]
Comment (jim-minter) [reply deploy/rp-development.json]: service vault contains the first party secret and TLS serving secret, accessible only via the RP identity.
cluster vault contains the cluster TLS secrets, accessible only via the FP identity.
I think the separation helps protect against accidentally giving a cluster a service secret.

File: pkg/install/2-removebootstrap.go (hunk 1 above)
Comment (jim-minter) [reply pkg/install/2-removebootstrap.go]: +1.

File: deploy/rp-development.json (hunk 2 above)
Comment (jim-minter) [reply deploy/rp-development.json]: v7.0 is the latest API by which you talk to a key vault (get key, create certificate, etc.)
2016-10-01 is (I think) the latest ARM API by which you create or configure a key vault

**Comment by jim-minter** [comment]
@asalkeld @julienstroheker @mjudeikis @m1kola this is pretty close now - a review from one of you would be appreciated

**Pull request merged**